tutorials/README.md

```diff
@@ -11,12 +12,14 @@
 | fileToJdbc        | Shows how to read files from HDFS, parse into POJOs and then insert into a table in MySQL.  |
 | hdht              | Shows how to use the HDHT operator. |
 | hdfs2kafka        | Shows how to read from HDFS and write to a Kafka topic. |
+| jdbcIngest        | Shows how to read rows from a table in an SQL database, polling and non-polling fashion, and write them to a file in HDFS. |
+| innerjoin         | Shows how to use streaming innerjoin operator|
 | jdbcIngest        | Shows how to read rows from a table in an SQL database and write them to a file in HDFS |
```

Comment:
Hmmm... this was from before my changes but I'll fix that. BTW the content of these lines are different but I'll remove the 2nd one as you suggest.